charts/enterprise-gateway/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Enterprise Gateway image name and tag to use.
+image: elyra/enterprise-gateway:2.3.0
+# Enterprise Gateway image pull policy.
+imagePullPolicy: IfNotPresent
+# The primary port on which Enterprise Gateway is servicing requests.
+port: 8888
+# Update to deploy multiple replicas of EG.
+replicas: 1
+# Log output level.
+logLevel: DEBUG
+# Whether to mirror working directories.
+mirrorWorkingDirs: false
+# Master public IP on which to expose EG.
+k8sMasterPublicIP:
+
+kernel:
+  # Kernel cluster role created by this chart.
+  clusterRole: kernel-controller
+  # Will start kernels in the same namespace as EG if True.
+  shareGatewayNamespace: false
+  # ignored if shareGatewayNamespace
+  kernelNamespace: spark-notebooks
```

Comment:
Currently we also have non-Spark kernels avaiable to the user (vanilla python, vanilla R, vanilla Scala, and Python with Tensorflow).  I think we should just call the namespace `notebooks`.

Perhaps we should restrict the available kernels to only Spark kernels, I'm not sure (the `whitelist` configuration value below).